dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -130,6 +130,16 @@ UpdateLanStatus(HWND hwndDlg,  LANSTATUSUI_CONTEXT * pContext)
         return;
     }
 
+    if(pContext->Status == (UINT)-1)
+    {
+        /*
+        On first execution, pContext->dwIn{out}Octets will be null while IF data are already refreshed with non null data so a gap is normal
+        and does not corresponds to an effective packet tx or rx
+        */
```

Comment:
```suggestion
        /*
         * On first execution, pContext->dwIn{out}Octets will be null while IF data is already refreshed
         * with non-null data, so a gap is normal and does not correspond to an effective tx or rx packet.
         */
```

Reply:
Moreover:
* dwIn{out}Octets: it's In and Out, not In and Inout.
* null: '0' rather.

Reply:
OK